bsmetadata/preprocessing_utils.py

```diff
@@ -182,62 +183,58 @@ def __init__(self, base_url, path_wiki_db):
             "mode": "eval",
             "model_path": "ed-wiki-2019",
         }
-        self.model = EntityDisambiguation(self.base_url, self.wiki_version, self.config)
+        self.model = EntityDisambiguation(self.base_url, self.wiki_version, self.config, reset_embeddings=True)
```

Comment:
I remember that you said to me that this argument will speed the code, can you give more info about what it does ? :slightly_smiling_face:

Reply:
According to this [PR](https://github.com/informagi/REL/issues/52), the performance improved a bit when `reset_embeddings` was set to `True`. 

Since the embeddings are stored in memory, the performance might become slower as the iterations progress and the memory becomes full. Setting `reset_embeddings` to `True`. resets the embeddings every time we run [get_data_items()](https://github.com/informagi/REL/blob/2bf75038729d54c79d57b8e7d5eac3d50c583690/REL/entity_disambiguation.py#L855) function in the `EntityDisambiguation` class and might improve the performance by a bit.